index.html

```diff
@@ -4,6 +4,7 @@
   <title>YOUR NAME HERE</title>
 </head>
 <body>
-
+<h1>Lotus</hi>
```

Comment:
Fix indentation.  Also your heading closing tag should be `</h1>`, not `</hi>`. 

Change to the following:
```html
<body>
  <h1>Lotus</h1>
  <img src="https://media.giphy.com/media/xT9DPmOJ1T72sStROo/giphy.gif" />
</body>